hs_core/management/utils.py

```diff
@@ -15,9 +15,12 @@
 import os
 import time
 
+from django.apps import apps
 from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
```

Comment:
Bases on the error I was getting on my fix for migration:
Remove this import

Reply:
did that in https://github.com/hydroshare/hydroshare/pull/5191/commits/d61680e7597996d4243fdb209a8e11a42ae60cdc thanks